meetings/2024-10-09.md

```diff
@@ -30,15 +39,111 @@
 
 
 ### Scribes
-* [name]()
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
 
 ### Introductions
-*
+* Niko: Developer of NextGraph, based on semantic web. Local-first and end-to-end encrypted.
+* Damon: Co founder of WeDistribute, indymedia org, covering decentralized social tech. Lead project manager behind memory project, building on top of ActivityPods.
 
 ---
 
 ## Topics
 
-### Use cases (continue from last week)
+### NextGraph overview
+
+URL: https://docs.nextgraph.org/en/introduction/
+
+* eP: Niko who is developing NextGraph will give 15-20min presentation
+* Niko: presenting slides and demo
+* ...: Peer to Peer semantic we collaboration
+* ...: NextGraph is doing the same as Solid but differently
+* ...: RDF, Linked Data, SPARQL
+* ...: Local-first CRDT of RDF - you can edit offline and data gets synchronized once you get online and without conflicts. I developed CRDTs for RDF, there are already formats for JSON. I added compatibility with automerge and yjs. This way one can mix all kinds of data. 
+* ...: end-to-end encrypted
+* ...: decentralized - 2 tier. Not peer-to-peer we have servers which act as brokers. It is based on pub/sub where replicas subscribe. All the changes are propageted. Brokers are constantly exchanging updates in anticipation for the devices that subscribe to them. Local replica offers SPARQL locally to the applictions.
+* ...: also binary data
+* ...: we have an app mobile, desktopand web - alpha
+* ...: a framework soon, Rust/JS&TS/Svelte
+* ...: a new protocol (binary) not using HTTP, no domain names and certs, also for efficiency
+* ...: we are collaborating with ActivityPods, already mentioned today, yesterday we annonced the collabortion. On Monday activitypods released v2.0 - it combines ActivityPub with Solid.
+* ...: ActivityPods also uses SPARQL, solid documents are named graphs. Next Graph is doing the same which fits well. They will also adopt DIDs for documents and agents, as NextGraph. A special DID method will be registered.
+* ...: Activity Pods are using SAI - Solid Application Interop, NextGraph is also using it.
+* ...: long term goal is unify our frameworks
+* ...: reactive stores in React, Svelte, Vue
+* ...: NextGraph has documents which are RDF resources, which have primary class.
+* ...: viewers / editors are the apps, manifest specifies which primary classes it can read and write
+* ...: end user can choose which viewer / editor it uses, user is the owenr of the data
+* ...: plans for everything app, to give one good app, there can be other apps available but we want to have one good app on every domain.
+* ...: it is also a social network thanks to pub/sub, it can be compatible with activitypub, also can be bridged to other protocols
+* ...: data is based on revisions, each change is a commit, one can come back to any past revision
```

Comment:
```suggestion
* ...: plans for "everything" app, to give one good app; there can be other apps available, but we want to have one good app on every domain.
* ...: it is also a social network. Thanks to pub/sub, it can be compatible with activitypub, and can also be bridged to other protocols.
* ...: data is based on revisions. Each change is a commit; one can come back to any past revision.
```